src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannelFactory.cs

```diff
@@ -450,8 +482,7 @@ protected override void OnEndClose(IAsyncResult result)
 
             protected internal override Task OnCloseAsync(TimeSpan timeout)
```

Comment:
Do you have a concrete recommendation @mconnew?  Many of these are overrides, and some of these now appear to be dead code. The OnCloseAsync appears to be the only real path executed, even for synchronous closes.  Would it be satisfactory to open another issue to deal with dead code removal and refactoring?  I would prefer not to refactor what is suspected to be dead code.